cluster-scope/base/core/namespaces/kepler/namespace.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: monitoring
```

Comment:
I see that this is where all the kepler resources are being deployed, but this name is very generic and I think it might be a little confusing. Is this a required name for this namespace? can it be something like `kepler-monitoring` ?

Reply:
the folder name here `/core/namespace/<your-namespace>`

the `<your-namespace>` should match the `metadata.name` of the namespace